server.js

```diff
@@ -1952,6 +1952,171 @@ function objectWebServer() {
         res.sendStatus(200); // OK
     });
 
+    function parseCategory(result) {
```

Comment:
This addition would be better organized if it were in its own file. One example of this is objectRouter and other routers defined above